common-ui/src/main/res/values/design-system-theming.xml

```diff
@@ -15,40 +14,52 @@
+        <item name="textAppearanceBody1">@style/TextAppearance.DuckDuckGo.Body1</item>
+        <item name="textAppearanceBody2">@style/TextAppearance.DuckDuckGo.Body2</item>
+        <item name="textAppearanceButton">@style/TextAppearance.DuckDuckGo.Button1</item>
+        <item name="textAppearanceButton1">@style/TextAppearance.DuckDuckGo.Button1</item>
+        <item name="textAppearanceButton2">@style/TextAppearance.DuckDuckGo.Button2</item>
+        <item name="textAppearanceCaption">@style/TextAppearance.DuckDuckGo.Caption</item>
 
-        <item name="android:windowBackground">?attr/colorPrimaryDark</item>
-        <item name="android:windowActivityTransitions">true</item>
-        <item name="android:statusBarColor">?toolbarBgColor</item>
-        <item name="windowActionBar">false</item>
-        <item name="windowNoTitle">true</item>
-        <item name="android:colorEdgeEffect">@color/newBlack</item>
+        <item name="textAppearanceTitle">@style/Typography.DuckDuckGo.Title</item>
+        <item name="textAppearanceH1">@style/Typography.DuckDuckGo.H1</item>
+        <item name="textAppearanceH2">@style/Typography.DuckDuckGo.H2</item>
+        <item name="textAppearanceH3">@style/Typography.DuckDuckGo.H3</item>
+        <item name="textAppearanceH4">@style/Typography.DuckDuckGo.H4</item>
+        <item name="textAppearanceH5">@style/Typography.DuckDuckGo.H5</item>
 
-        <!-- Old System Components (will be refactored soon) -->
+        <!-- Android Components -->
         <item name="actionOverflowMenuStyle">@style/PopUpOverflowMenu</item>
         <item name="actionOverflowButtonStyle">@style/PopUpOverflowButton</item>
-        <item name="snackbarButtonStyle">@style/snackbarButtonStyle</item>
-        <item name="alertDialogTheme">@style/AlertDialogTheme</item>
-        <item name="android:alertDialogTheme">@style/AlertDialogTheme</item>
+        <item name="bottomSheetDialogTheme">@style/Widget.DuckDuckGo.BottomSheetDialog</item>
+        <item name="tabStyle">@style/Widget.DuckDuckGo.TabLayout</item>
         <item name="radioButtonStyle">@style/Widget.DuckDuckGo.RadioButton</item>
-        <item name="autoCompleteTextViewStyle">@style/AutoCompleteTextViewStyle</item>
         <item name="sliderStyle">@style/Widget.DuckDuckGo.Slider</item>
         <item name="switchStyle">@style/Widget.DuckDuckGo.Switch</item>
-
-        <!-- Design System Themes -->
-        <item name="bottomSheetDialogTheme">@style/Widget.DuckDuckGo.BottomSheetDialog</item>
+        <item name="snackbarButtonStyle">@style/Widget.DuckDuckGo.Snackbar</item>
+        <item name="autoCompleteTextViewStyle">@style/Widget.DuckDuckGo.AutoCompleteTextView</item>
+        <item name="alertDialogTheme">@style/AlertDialogTheme</item>
+        <item name="android:alertDialogTheme">@style/AlertDialogTheme</item>
+        <item name="materialAlertDialogTheme">@style/ThemeOverlay.MaterialComponents.Dialog.Alert</item>
```

Comment:
Dialogs built with `MaterialAlertDialogBuilder` don't look good in dark mode. Setting `materialAlertDialogTheme` to `@style/AlertDialogTheme` will fix them
```suggestion
        <item name="materialAlertDialogTheme">@style/AlertDialogTheme</item>
```